{
  "type": "unknown",
  "name": "FileAlreadyExistsException",
  "signature": "class FileAlreadyExistsException : FileSystemException",
  "items": [
    {
      "text": "FileAlreadyExistsException( file: File, other: File? = null, reason: String? = null)",
      "desc": "An exception class which is used when some file to create or copy to already exists."
    },
    {
      "text": "val Throwable.stackTrace: Array<StackTraceElement>",
      "desc": "Returns an array of stack trace elements representing the stack trace pertaining to this throwable."
    },
    {
      "text": "fun Throwable.printStackTrace(writer: PrintWriter)",
      "desc": "Prints the detailed description of this throwable to the specified writer."
    },
    {
      "text": "fun Throwable.printStackTrace(stream: PrintStream)",
      "desc": "Prints the detailed description of this throwable to the specified stream."
    }
  ]
}